src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/THM_wellbore_stress.py

```diff
@@ -0,0 +1,146 @@
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+import numpy as np
+import h5py
+from numpy import genfromtxt
+import analyticalResults
+
+def stressRotation(stress, phi_x):
+    rotx = np.array([[np.cos(phi_x), np.sin(phi_x), 0.], [-np.sin(phi_x), np.cos(phi_x), 0.], [0., 0., 1.]])
+    return np.dot(np.dot(np.transpose(rotx), stress), rotx)
+
+def main():
+	# Material properties from input XML file
+	bulkModulus = 20.7e9 # drained bulk modulus of the porous medium
+	thermalExpansionCoefficients = 4e-5 # drained linear thermal expansion coefficient of the porous medium
+	grainBulkModulus = 23.5e9
```

Comment:
If you want, you can read the xml in Python and get these values, it will be much cleaner: https://github.com/GEOSX/GEOSX/blob/4de9026c24f539c8354c396c35ae8b422008448f/src/docs/sphinx/advancedExamples/validationStudies/faultMechanics/mandel/mandelFigure.py#L74

Reply:
done